packages/inter-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -495,7 +495,6 @@ const publicBehavior = {
   makeLoanInvitation: makeVaultInvitation,
   /** @deprecated use getCollateralManager and then makeVaultInvitation instead */
   makeVaultInvitation,
-  getCollaterals,
```

Comment:
nit: I wouldn't expect a breaking change in a chore. Perhaps this is in some sense a fix? Perhaps there's a problem that motivates the change?

Reply:
or a feat that motivates it, such as durable publish kits. In that case, the breaking change flag should (also?) go on the feat/fix.

I don't think conventional commit changelog tools pick up chores, do they?

Reply:
Chore commit do show up. This commit does have the `!` to note a breaking change. https://github.com/Agoric/agoric-sdk/pull/6772/commits/48e6d5dddd01044727c9f06a8a1520e563d6525b

I can change the commit message to whatever you request.

To the question of whether it's legitimate to break this: @samsiegart do you need it? I see it in dapp-treasury but not in dapp-inter. (Those are the only two consumers I think matter as this functionality has never been released)

Reply:
Nothing on the UI side needs it, no

Reply:
No particular request. Just floating some ideas. Feel free to take them or leave them.

Reply:
Welp, https://github.com/Agoric/testnet-load-generator/blob/8eb7e6cb67e86fa3c968a9c0e0d1882eeb21736b/loadgen/contract/agent-create-vault.js#L78 expects this on the publicFacet so I'm keeping that one and dropping the `machine` one